### ISSUE TITLE
UI Update: Fix Column Titles on "Transactions in this report" Page

### ISSUE THREAD

**Issue by MitchellTCG** [fecgov/fecfile-web-app#1254]
Business Reason
As Principal UX, I would like the titles of the columns to match the wireframes

 Task
Update Receipt and Disbursement sections
- [x] 1. Change "Transaction type" to "Type"
- [x] 2. Change "Memo code" to "Memo"

 Wireframe
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcWpoIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--f0d5c92bd7ef178ab177268410d9191321e68193/1057%20-%20Transactions%20in%20this%20report%20%E2%80%93%20Loan%20%E2%80%93%202.png)

### QA Notes ### 
 null
### DEV Notes ### 
 null
### Design ### 
 null

**Comment by mjtravers** [comment]
Passes CR. Sending to QA.

**Comment by WiseQA** [comment]
QA review verified the "Transactions" in this report" table column titles for Receipt and Disbursement sections we updated as follows:

 1. Change "Transaction type" > "Type"
 2. Change "Memo code" > "Memo"'
 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdFQ4IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--d7d669d83dbb7b0cfb31f2c208d8291bf7e460bb/image.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdGY4IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--923412cc4d24ebbba4ab8f6eae59c969570c14ac/image.png)

QA Review Completed.  Moved to Stage Ready.
